docs/dev/building-on-zksync/useful-address.md

```diff
@@ -90,9 +90,9 @@ _Users should only interact with the token bridge contracts via [zkSync Bridge](
 
 ## Known tokens
 
-|    Token Name                 | Symbol        | L1 token address                          | L2 token address                         |
+|    Symbol                     | Name        | L1 token address                          | L2 token address                         |
 | ----------------------------- | ------------- | ------------------------------------------| ------------------------------------------
-| Mute.io                       | MUTE          | [0xa49d7499271ae71cd8ab9ac515e6694c755d400c](https://etherscan.io/address/0xa49d7499271ae71cd8ab9ac515e6694c755d400c)|[0x0e97C7a0F8B2C9885C8ac9fC6136e829CbC21d42](https://explorer.zksync.io/address/0x0e97C7a0F8B2C9885C8ac9fC6136e829CbC21d42)|
-| Furucombo                     | COMBO         | [0xffffffff2ba8f66d4e51811c5190992176930278](https://etherscan.io/address/0xffffffff2ba8f66d4e51811c5190992176930278)|[0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3](https://explorer.zksync.io/address/0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3#contract)|
-| Perpetual                     | PERP          |  [0xbC396689893D065F41bc2C6EcbeE5e0085233447](https://etherscan.io/address/0xbC396689893D065F41bc2C6EcbeE5e0085233447)|[0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601](https://explorer.zksync.io/address/0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601#contract)|
-| USD Coin                      | USDC          | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
+| MUTE                          | Mute.io       | [0xa49d7499271ae71cd8ab9ac515e6694c755d400c](https://etherscan.io/address/0xa49d7499271ae71cd8ab9ac515e6694c755d400c)|[0x0e97C7a0F8B2C9885C8ac9fC6136e829CbC21d42](https://explorer.zksync.io/address/0x0e97C7a0F8B2C9885C8ac9fC6136e829CbC21d42)|
+| COMBO                         | Furucombo     | [0xffffffff2ba8f66d4e51811c5190992176930278](https://etherscan.io/address/0xffffffff2ba8f66d4e51811c5190992176930278)|[0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3](https://explorer.zksync.io/address/0xc2B13Bb90E33F1E191b8aA8F44Ce11534D5698E3#contract)|
+| PERP                          | Perpetual     |  [0xbC396689893D065F41bc2C6EcbeE5e0085233447](https://etherscan.io/address/0xbC396689893D065F41bc2C6EcbeE5e0085233447)|[0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601](https://explorer.zksync.io/address/0x42c1c56be243c250AB24D2ecdcC77F9cCAa59601#contract)|
+| USDC                          | USD Coin      | [0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48)|[0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4](https://explorer.zksync.io/address/0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4#contract)|
```

Comment:
@githubdoramon Isn't this suppose to be Circle, as the name?

Reply:
Circle is the company... the token is USD Coin